var/spack/repos/builtin/packages/libjpeg-turbo/package.py

```diff
@@ -34,12 +34,24 @@ class LibjpegTurbo(AutotoolsPackage):
     homepage = "http://libjpeg-turbo.virtualgl.org"
     url      = "http://downloads.sourceforge.net/libjpeg-turbo/libjpeg-turbo-1.3.1.tar.gz"
 
+    version('1.5.1', '55deb139b0cac3c8200b75d485fc13f3')
     version('1.5.0', '3fc5d9b6a8bce96161659ae7a9939257')
+    version('1.4.2', '86b0d5f7507c2e6c21c00219162c3c44')
     version('1.3.1', '2c3a68129dac443a72815ff5bb374b05')
 
+    variant('java', default=False, description='Enable Java build')
+
+
     # Can use either of these. But in the current version of the package
     # only nasm is used. In order to use yasm an environmental variable
     # NASM must be set.
     # TODO: Implement the selection between two supported assemblers.
     # depends_on("yasm", type='build')
     depends_on("nasm", type='build')
+    
+    def configure_args(self):
+        args = []
+        if '+java' in self.spec:
+            args.append('--with-java')
```

Comment:
You might want to add:
```python
else:
    args.append('--without-java')
```
Otherwise you could end up with a non-deterministic build. Someone could build `libjpeg-turbo~java` but end up with Java support because it picked up a system java installation.